README.md

```diff
@@ -179,7 +179,8 @@ First grab the sources and run the setup script:
 ```console
 git clone https://github.com/tigerbeetle/tigerbeetle.git
 cd tigerbeetle
-scripts/install.sh
+./scripts/install_zig.sh
```

Comment:
Ah! Let's point the user to the quick start instructions above? They no longer need source to benchmark.

Reply:
I removed this section altogether. I don't think it's possible to meaningfully use these benchmarks unless you know quite a bit more than we are saying here.